config/app.config

```diff
@@ -21,62 +21,56 @@ ehcache.multicast.ttl = 1
-help.base.url = "http://irsa.ipac.caltech.edu/onlinehelp/"
+sso.server.url = "https://irsa.ipac.caltech.edu/account/"
+sso.user.profile.url = "https://irsa.ipac.caltech.edu/account/uman/uman.html//id=profile"
+help.base.url = "https://irsa.ipac.caltech.edu/onlinehelp/"
 
 
 visualize.fits.MaxSizeInBytes= 10737418240
 visualize.fits.search.path = "/irsadata"
+visualize.fits.Security= true
+
+// ehcache.xml env sensitive properties
+// ehcahe replication port; suggest 4015-local, 5015-dev, 6015-test, 7015-ops, 7515-ops_int
+ehcache.multicast.port = "7015"
+
 
-irsa.gator.hostname = "irsa.ipac.caltech.edu"
+/* ------------------------ IRSA services --------------------------------- */
+GatorHost     = "https://irsa.ipac.caltech.edu"
+irsa.base.url           = "https://irsa.ipac.caltech.edu"
+irsa.gator.dd.hostname  = "https://irsa.ipac.caltech.edu"
+irsa.gator.hostname     = "https://irsa.ipac.caltech.edu"
+//MOST (moving object - precovery - search) host
+most.host = "https://irsasearchops.ipac.caltech.edu/cgi-bin/MOST/nph-most"
+// IRSA Periodogram API
+irsa.gator.service.periodogram.url = "https://irsa.ipac.caltech.edu/cgi-bin/periodogram/nph-periodogram_api"
+/*  ----------------------------------------------------------------------- */
 
 wise.ibe.host       = "irsa.ipac.caltech.edu/ibe"
 twomass.ibe.host    = "irsa.ipac.caltech.edu/ibe"
 
 lsst.dataAccess.uri = ''
 lsst.dataAccess.db = ''
 
-// not used at the moment - since Aug. 2015 - may come back (check Git history on QueryLSSTCatalog)
-// lsst.qserv.uri = ''
-// lsst.qserv.user = ''
-// lsst.qserv.pass = ''
-
-// not used at the moment - since Aug. 2015 - may come back (check Git history on QueryLSSTCatalogDD)
-// lsst.schema.uri = ''
-// lsst.schema.user = ''
-// lsst.schema.pass = ''
-
-
-// ehcache.xml env sensitive properties
-// ehcahe replication port; suggest 4016-developer, 5016-dev, 6016-I&T, 7016-Prod, 8016-Public
-ehcache.multicast.port = 4016
-visualize.fits.Security= true
-
-// IRSA Periodogram API
-irsa.gator.service.periodogram.url = "http://irsatest.ipac.caltech.edu/cgi-bin/periodogram/nph-periodogram_api"
-
-environments{
+environments {
     local {
-        visualize.fits.search.path = "/Library/WebServer/Documents:/irsadata:/hydra"
+        BuildType = "Development"
+        ehcache.multicast.port = 4015
         visualize.fits.Security= false
-        visualize.fits.MaxSizeInBytes= 10737418240
         ehcache.multicast.ttl = 0
     }
-
     dev {
+        BuildType = "Development"
         ehcache.multicast.port = "5015"
+        irsa.gator.service.periodogram.url = "https://irsadev.ipac.caltech.edu/cgi-bin/periodogram/nph-periodogram_api"
         lsst.dataAccess.uri = "http://localhost:8661/db/v0/query?"
         lsst.dataAccess.db = "smm_bremerton"
     }
-
     test {
-      BuildType = "Beta"
-      ehcache.multicast.port = "6015"
+        BuildType = "Beta"
+        ehcache.multicast.port = "6015"
```

Comment:
I think the periodogram URL is missing in test env, right?
irsa.gator.service.periodogram.url = "https://irsatest.ipac.caltech.edu/cgi-bin/periodogram/nph-periodogram_api